test/client/fluentd/fluentd_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package fluentd_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestFluentd(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Fluentd Suite")
```

Comment:
IIRC correct we configure the junit reporter for E2E tests. I suggest to consider the similar ginkgo register routines in our E2E test suite.

Reply:
+1